OpenRA.Mods.Common/Traits/Captures.cs

```diff
@@ -142,11 +141,10 @@ public override bool CanTargetActor(Actor self, Actor target, TargetModifiers mo
 
 			public override bool CanTargetFrozenActor(Actor self, FrozenActor target, TargetModifiers modifiers, ref string cursor)
 			{
-				// Actors with FrozenUnderFog should not disable the Capturable trait.
-				var c = target.Info.TraitInfoOrDefault<CapturableInfo>();
-				if (c == null || !c.CanBeTargetedBy(self, target.Owner))
+				var captureManagerInfo = target.Info.TraitInfoOrDefault<CaptureManagerInfo>();
+				if (captureManagerInfo == null || !captureManagerInfo.CanBeTargetedBy(target, self, captures))
```

Comment:
I've been wondering about this with `ConditionManager` already:

Considering that the *Manager traits are neither conditional nor do the logics using them work when they're missing, wouldn't it actually be more modder-friendly to drop the null checks and crash if they're missing?
The way I see it, in cases like this modders will wonder why capturing doesn't work, instead of getting an exception that tells them that CaptureManager is missing.

Reply:
If this is caught by the linter and optionally mod startup, then I guess this would be fine.